.eslintrc.js

```diff
@@ -347,8 +362,15 @@ module.exports = {
     {
       files: ['ui/pages/settings/*.js'],
       rules: {
-        'sort-keys': ['error', 'asc', { natural: true }],
+        'sort-keys': [
+          'error',
+          'asc',
+          {
+            natural: true,
+          },
+        ],
       },
     },
   ],
+  extends: ['plugin:storybook/recommended'],
 };
```

Comment:
Mostly lint fixes? But this is the added plugin from storybook

Reply:
Great suggestion have updated in https://github.com/MetaMask/metamask-extension/pull/15684/commits/6a7a0cc9987c354e0846bfbb3f298688a70e466b